src/plugins/kdb/ldap/libkdb_ldap/ldap_misc.c

```diff
@@ -1069,6 +1117,20 @@ krb5_add_int_mem_ldap_mod(LDAPMod ***list, char *attribute, int op, int value)
 }
 
 krb5_error_code
+krb5_ldap_modify_ext(krb5_context context, LDAP *ld, const char *DN,
+                     LDAPMod **mods, int op)
+{
+    krb5_error_code ret;
+
+    ret = ldap_modify_ext_s(ld, DN, mods, NULL, NULL);
+    if (ret != LDAP_SUCCESS) {
+        ret = set_ldap_error(context, ret, op);
+    }
+
+    return ret;
+}
```

Comment:
The DN argument should be named "dn", not "DN".  This function can be shortened and made more correct (using the correct type for the return valid of ldap_modify_ext_s()) as follows:

    int st;

    st = ldap_modify_ext_s(ld, dn, mods, NULL, NULL);
    return (st == LDAP_SUCCESS) ? 0 : set_ldap_error(context, st, op);